messages/protos/messages.proto

```diff
@@ -622,6 +637,35 @@ message MegaphoneEnabledMessage {
   bool value = 1;
 }
 
+message MuteParticipantIdSpaceMessage {
+  string spaceName = 1;
+  string value = 2;
+}
+
+message MuteEveryBodySpaceMessage {
+  string spaceName = 1;
+  string userId = 2;
```

Comment:
Maybe rename to `notMutedUserUuid` for additional clarity?

Let's ALWAYS use "uuid" if this is a string, and "id" if this is the number.

Reply:
or senderUserId maybe?